src/components/ec-timer/ec-timer.story.js

```diff
@@ -37,3 +37,21 @@ stopped.args = {
 stopped.parameters = {
   visualRegressionTests: { disable: true },
 };
+
+export const stoppedWithMinutes = Template.bind({});
+stoppedWithMinutes.args = {
+  seconds: 30,
+  showMinutes: true,
+  isRunning: false,
+};
+
+stoppedWithMinutes.parameters = {
+  visualRegressionTests: { disable: true },
+};
+
+export const showMinutes = Template.bind({});
+showMinutes.args = {
+  showMinutes: true,
+  seconds: 80,
+  isRunning: true,
+};
```

Comment:
shouldn't we disable regression tests for this story?

Reply:
why should we?

Reply:
I guess we disable tests for dynamically changed stories, and here we have timer running.